src/Symfony/Component/Messenger/DependencyInjection/MessengerPass.php

```diff
@@ -252,6 +252,15 @@ private function registerReceivers(ContainerBuilder $container, array $busIds)
                 ->replaceArgument(4, $busIds);
         }
 
+        if ($container->hasDefinition('console.command.messenger_setup_transports')) {
+            $receiverNames = [];
+            foreach ($receiverMapping as $name => $reference) {
+                $receiverNames[(string) $reference] = $name;
+            }
```

Comment:
We have these same 4 lines further up. I think we should just run this before both `if` statements. NOT having these definitions is a serious edge-case anyways (only applies to people not using the FWBundle but still using the compiler passes) so it's ok to run that logic even if someone might not need it.